src/main/webapp/css/srp_style.css

```diff
@@ -37,33 +37,68 @@ h1 {
 
 .card {
   margin-bottom: 20px;
-  padding: 24px;
+  padding: 24px 16px 24px 24px;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  max-height: 260px;
+  overflow: scroll;
+  font: 16px Roboto, Arial, sans-serif;
+  margin-top: 20px;
```

Comment:
You already have `margin-bottom`. Maybe replace with `margin: 20px 0`? (Put whatever margin property(s) with padding so similar properties are near each other.)